src/mrpro/algorithms/_remove_readout_os.py

```diff
@@ -31,49 +34,46 @@ def remove_readout_os(kdata: KData) -> KData:
 
     Returns
     -------
-        K-space data with oversampling removed.
+        (A new copy of) K-space data with oversampling removed.
```

Comment:
Copy of k-space data

The brackets might suggest that this only happens sometimes and a copy is usually something new.